src/meshes/p4est_mesh_view.jl

```diff
@@ -156,13 +155,18 @@ function calc_node_coordinates!(node_coordinates,
 
     trees = unsafe_wrap_sc(p4est_tree_t, mesh.parent.p4est.trees)
 
-    for tree_view in eachindex(mesh.cell_ids)
-        tree = mesh.cell_ids[tree_view]
-        offset = trees[tree_view].quadrants_offset
-        quadrants = unsafe_wrap_sc(p4est_quadrant_t, trees[tree].quadrants)
+    mesh_view_cell_id = 0
+    for tree_id in eachindex(trees)
+        tree_offset = trees[tree_id].quadrants_offset
+        quadrants = unsafe_wrap_sc(p4est_quadrant_t, trees[tree_id].quadrants)
 
         for i in eachindex(quadrants)
-            element = offset + i
+            parent_cell_id = offset + i
+            if !(parent_cell_id in mesh.cell_ids)
```

Comment:
```suggestion
            parent_mesh_cell_id = offset + i
            if !(parent_mesh_cell_id in mesh.cell_ids)
```
Sorry, I just realized that I gave a rather bad name to this variable. "parent cell" in hierarchical meshes has already a very precise, fixed meaning, thus we should make sure the user understands that we are referring to the parent mesh here.